lib/routes/bupt/scss.ts

```diff
@@ -0,0 +1,119 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import timezone from '@/utils/timezone';
+import { parseDate } from '@/utils/parse-date';
+import type { Context } from 'hono';
+
+export const route: Route = {
+    path: '/scss/:type',
+    categories: ['university'],
+    example: '/bupt/scss/xwdt',
+    parameters: {
+        type: {
+            type: 'string',
+            optional: false,
+            description: '信息类型，可选值：新闻动态，通知公告',
```

Comment:
`type` key does not exist in https://github.com/DIYgod/RSSHub/blob/89fe12969f3002352e9ba0ca86d979bde9d4ee68/lib/types.ts#L296-L302